mitosheet/src/mito/components/endo/celleditor/CellEditorDropdown.tsx

```diff
@@ -210,6 +211,14 @@ const CellEditorDropdown = (props: {
                     </div>
                 </div>
             }
+            {/* Always display a link to the documentation */}
+            <a
+                className='text-body-2 text-body-2-link cell-editor-dropdown-link'
```

Comment:
btw, we created a utility that helps us avoid missing space bugs in class names. The function is called `classNames` and it just takes each of these strings as arguments and returns this exact string. 

It also lets you pass dictionaries to it with the format {<class name> : <True or False>} so that you can conditionally apply css classes.